compose/src/main/java/com/bottlerocketstudios/compose/widgets/Text.kt

```diff
@@ -0,0 +1,37 @@
+package com.bottlerocketstudios.compose.widgets
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material.Icon
+import androidx.compose.material.LocalContentColor
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+
+@Composable
+fun IconText(
```

Comment:
@ianslinker-br @BottleRocket-Colin How do you feel about having previews for building block Composables like this one (likely in the same file)?

Reply:
Ah whoops. I had been adding them for thee other widgets I pulled out into separate files but forgot to add one here. Will add one now.